.pre-commit-config.yaml

```diff
@@ -22,3 +22,13 @@ repos:
   rev: 2.1.4
   hooks:
     - id: shellcheck
+- repo: local
+  hooks:
+    - id: avoid-short-arguments
+      name: avoid-short-arguments
+      description: >-
+        Prevent usage of short arguments, which significantly hamper
+        readability and compatibility.
+      language: pygrep
+      entry: ' [-]A'
+      files: '^.*$'
```

Comment:
Is this really a good check for this? Is there some situation where this could caught false positives? This also doesn't catches the case where the arguments are in quotes, as in:
```
append([]string{"delete", resource, "-A", "--all"}, extraArgs...)
```
right?
And since that's probably the structure most code will have I don't see the greater good of this specific grep